apps/recnet/src/clientEnv.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from "zod";
+
+export const clientEnvSchema = z.object({
+  NEXT_PUBLIC_FIREBASE_API_KEY: z.string(),
+  NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN: z.string(),
+  NEXT_PUBLIC_FIREBASE_PROJECT_ID: z.string(),
+  NEXT_PUBLIC_FIREBASE_APP_ID: z.string(),
+  NEXT_PUBLIC_GA_TRACKING_ID: z.string(),
+});
+
+const clientEnvRes = clientEnvSchema.safeParse({
+  NEXT_PUBLIC_FIREBASE_API_KEY: process.env.NEXT_PUBLIC_FIREBASE_API_KEY,
+  NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN:
+    process.env.NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN,
+  NEXT_PUBLIC_FIREBASE_PROJECT_ID: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
+  NEXT_PUBLIC_FIREBASE_APP_ID: process.env.NEXT_PUBLIC_FIREBASE_APP_ID,
+  NEXT_PUBLIC_GA_TRACKING_ID: process.env.NEXT_PUBLIC_GA_TRACKING_ID,
+});
+
+if (!clientEnvRes.success) {
+  // console.error(clientEnvRes.error.issues);
```

Comment:
[nit] do we need this?

Reply:
I think we should preserve this so that if other developers run the app on their local machine without proper `.env`, an error will be thrown to let them know. Plus, we can catch the env error when testing before merging into prod.